webapi/src/main/scala/org/knora/webapi/slice/admin/domain/model/DefaultObjectAccessPermissionRepo.scala

```diff
@@ -6,17 +6,51 @@
 package org.knora.webapi.slice.admin.domain.model
 
 import zio.Chunk
+import zio.NonEmptyChunk
 import zio.Task
 
+import org.knora.webapi.slice.admin.domain.model.DefaultObjectAccessPermission.DefaultObjectAccessPermissionPart
+import org.knora.webapi.slice.admin.domain.model.DefaultObjectAccessPermission.ForWhat
 import org.knora.webapi.slice.admin.domain.model.KnoraProject.ProjectIri
 import org.knora.webapi.slice.admin.repo.service.EntityWithId
 import org.knora.webapi.slice.common.repo.service.CrudRepository
+import org.knora.webapi.slice.resourceinfo.domain.InternalIri
 
 final case class DefaultObjectAccessPermission(
   id: PermissionIri,
   forProject: ProjectIri,
+  forWhat: ForWhat,
+  permission: Chunk[DefaultObjectAccessPermissionPart],
 ) extends EntityWithId[PermissionIri]
 
+object DefaultObjectAccessPermission {
+  enum ForWhat {
```

Comment:
Wouldn't ForWhat be like a SubjectEntity or something? (Entity being sort of a stand-in for a tuple.)

Reply:
I also noticed that you named it after the `forProject`.

I guess you can resolve this and merge, if you so please.